src/Sylius/Behat/Page/Admin/Product/Common/ProductMediaTrait.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin\Product\Common;
+
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+
+trait ProductMediaTrait
+{
+    /** @var string[] */
+    private array $imageUrls = [];
+
+    public function getDefinedProductMediaElements(): array
+    {
+        return [
+            'add-image' => '[data-test-add-image]',
```

Comment:
Use snake_case notation for keys
```suggestion
            'add_image' => '[data-test-add-image]',
```

Reply:
https://github.com/Sylius/Sylius/pull/16349/files#diff-fd91c9ddf0b416af72abdd28aad3bd674c01da691affeb9de436c9df50abfb3dR40